XIPs/xip-43-permission-pref-proofs.md

```diff
@@ -0,0 +1,91 @@
+
+## Specification
+
+There are 3 components to the proposed workflow:
+
+1. Obtain a permission signature from the user  
+    _Requires actions by the **sender** and **receiver**_
+2. Create a new conversation with the encoded payload  
+    _Requires actions by the **sender**_
+3. Verify the permission payload to allow the sender  
+    _Requires actions by the **client SDK**_
+
+### Obtain a permission signature from the user
+
+A permission signature must be obtained from a user's wallet, which is then encoded into a payload that can be used by client SDKs to verify and validate opt-in before allowing the sender.
+
+The decoded payload that must be collected by senders will be defined in a protobuf as follows:
+
+```protobuf
+message PermissionPayload {
+  // the user's signature in hex format
+  string signature = 1;
+  // approximate time when the user signed
+  uint64 timestamp = 2;
+  // version of the payload
+  uint32 payload_version = 3;
+}
+```
+
+The message that will be signed by the user's wallet will contain the sender's address and a timestamp. It must include a human-readable explanation, such as follows.
+
+```text
+XMTP : Grant inbox permission to sender
+
+Current Time: <ISO 8601 date and time in UTC>
+From Address: <ethereum address>
+
+For more info: https://xmtp.org/signatures/
+```
+
+A light-weight JavaScript bundle will provide a function that will initiate the signing process and return an encoded payload that senders must store on their end. This function is intended to be used as a callback to a click event, such as clicking on a Subscribe button.
+
+### Create a new conversation with the encoded payload
+
+Once senders have the encoded payload, they can include it when starting a new conversation with a user.
+
+An example of what this might look like:
+
+```ts
+const conversation = await client.conversations.newConversation(peerAddress, context, permissionPayload);
+```
+
+Users who have created a permission signature may not have an identity on the XMTP network. Senders should check for an identity with `Client.canMessage` prior to starting a new conversation. If a user does not yet have an XMTP identity, the sender can routinely check for a network identity and start a conversation when it's found.
+
+### Verify the permission payload
+
+In order to finalize the permission preference, SDKs must look for the permission payload in new conversations. Using this payload, SDKs can verify that the current user's wallet signed the permission message and validate that the addresses and timestamp match the expected values.
+
+Once the permission payload is verified and validated, the SDKs will then update network permission preferences automatically.
+
+## Backward Compatibility
+
+The permission signature is an _optional_ parameter when starting a new conversation. Existing conversations will not be affected, and client apps using outdated SDKs will continue to work without updates.
```

Comment:
the first sentence has been changed to: `The encoded permission payload is an _optional_ parameter when starting a new conversation.`